.github/workflows/test.yml

```diff
@@ -109,7 +109,7 @@ jobs:
         run: pip install -U tox
       - name: Cache tox environment
         # Preserves .tox directory between runs for faster installs
-        uses: actions/cache@v2
+        uses: actions/cache@v1
```

Comment:
Do we still need this after [this](https://github.com/open-telemetry/opentelemetry-python/blob/main/tox.ini#L143)?

Reply:
Fixed